private/dev/Dashboard.js

```diff
@@ -165,6 +165,11 @@ export default () => {
       uppyDashboard.use(Tus, { endpoint: TUS_ENDPOINT, limit: 6 })
       break
     case 's3':
+      uppyDashboard.use(AwsS3, {
+        endpoint: COMPANION_URL,
+      })
+      break
+    case 's3-non-multipart':
```

Comment:
This is not needed. Look at the existing code, `s3` = `shouldUseMultipart: false` and `s3-multipart` = `shouldUseMultipart: true`. You're adding a duplicate option.

Reply:
no, i was renaming the option s3 to s3-non-multipart, and i added a new option s3 which is dynamic (default option for shouldUseMultipart, meaning it [depends on file size](https://uppy.io/docs/aws-s3/#shouldusemultipartfile))

Reply:
If that's what you want you can just remove the option on either one for a brief moment and then don't commit that.